docs/concepts/configuration-documents/overview.md

```diff
@@ -0,0 +1,255 @@
+---
+description: >-
+  DSC configuration documents are YAML or JSON data files that define the desired state of a system
+  declaratively. They're used to retrieve, validate, and enforce the state of multiple resource
+  instances.
+ms.date: 03/25/2025
+title: DSC configuration documents
+---
+
+# DSC configuration documents
+
+<!-- markdownlint-disable MD013 -->
+
+In Microsoft's Desired State Configuration (DSC) platform, DSC configuration documents declare the
+desired state of a system as data files. Configuration documents define a collection of
+[DSC resource][01] instances to describe what the desired state should be, not how to put the
+system into that state. The DSC resources handle the _how_ for each instance.
+
+DSC can process configuration documents to:
+
+- Retrieve the current state of the defined resource instances with the `dsc config get` command.
+- Validate whether the instances are in the desired state with the `dsc config test` command.
+- Enforce the desired state for the instances with the `dsc config set` command.
+- Export a new configuration document with every instance of a set of resources with the
+  `dsc config export` command.
+
+Configuration documents are YAML or JSON files that contain a single object. The object's
```

Comment:
Does it help to mention relationship to ARM templates since those have more docs and examples?

Reply:
And given the very positive reaction to Bicep at PS Summit, this makes it sound like we _only_ support the two languages, when we actually support a third, as an abstraction of the JSON.